front/src/components/messaging/MessageList.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC } from 'react';
+import { ScrollView, StyleSheet, View } from 'react-native';
+
+import { Message } from '@/models/entities/message';
+
+import MessageBubble from './MessageBubble';
+
+/**
+ * The styles for the MessageList component.
+ */
+const styles = StyleSheet.create({
+  container: {
+    gap: 10,
+  },
+  messageBubbleContainer: {
+    marginBottom: 30,
+  },
```

Comment:
Can you exclusively use the `gap`?
```suggestion
  container: {
    gap: 10,
  },
```